.github/workflows/build.yml

```diff
@@ -139,7 +139,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-       macos-version:  ['10.15']
+       macos-version:  ['11']
```

Comment:
I think the actual problem is the missing support for Python 2.7, also with the previous runner. I'm not sure how to test Python 2.7 on MacOS at all with that problem...

Reply:
Ok, and I just saw:
> Warning: The support for python 2.7 will be removed on June 19

for the Windows build, so the same problem here :(